main.c

```diff
@@ -228,7 +228,44 @@ void remove_log(
     struct board_tile board[SIZE][SIZE],
     int last_coordinate[2]
 ) {
-    printf("// TODO remove_log [row] [column]\n");
+    int row, col;
+    printf ("To remove log(s), please input [row] [column]: ");
+    scanf ("%d %d", &row, &col);
+
+    //Looping to check if log tile is occupied by frogger. Log tile occupied by frogger will not be changed to water.
+    int i = 0;
+    while (i < SIZE) //Following the size of the board to check the whole row (adjacent possible logs to the left and right).
+    {
+        if (board[row][i].occupied) //Tile occupied by Frogger.
+        {
+            printf ("The log at row %d and column %d cannot be removed as Frogger is there.\n", row, col);
+            return; //Do nothing to the tile.
+        }
+        i++;
+    }
```

Comment:
we can simplify this part by checking from `last_coordinate`, similar to how clear row is implemented

```suggestion
    // Check if Frogger is on the row
    if (last_coordinate[X] == row) {
        printf("Error: Cannot clear row %d because Frogger is present.\n", row);
        return;
    }
```